examples/pod.yml

```diff
@@ -3,7 +3,12 @@ kind: Pod
 metadata:
   name: samplepod-1
   annotations:
-    k8s.v1.cni.cncf.io/networks: macvtap0
+    k8s.v1.cni.cncf.io/networks: '[
```

Comment:
Mind adding this as a second example `pod-with-specific-mac.yml` or something like that? Let's not scare anyone.

Reply:
Done.